api/src/pcapi/core/finance/models.py

```diff
@@ -1055,6 +1055,30 @@ def isClosed(cls) -> Exists:  # pylint: disable=no-self-argument
             .where(Cashflow.status == CashflowStatus.ACCEPTED)
         )
 
+    @property
+    def cashflow_batch_label(self) -> str | None:
+        for booking_finance_incident in self.booking_finance_incidents:
+            for finance_event in booking_finance_incident.finance_events:
+                for pricing in finance_event.pricings:
+                    if pricing.status in (PricingStatus.PROCESSED, PricingStatus.INVOICED):
+                        for cashflow in pricing.cashflows:
+                            return cashflow.batch.label
+        return None
```

Comment:
la fonction me semble assez fragile a moins que tout les bookings dans in incidents soient TOUJOURS dans le même batch. Sinon a chaque fois qu'on va l'appeller on aura potentiellement un résultat différent. (je ne connais pas assez le métier pour juger)